grails-app/i18n/messages_fr.properties

```diff
@@ -826,7 +826,7 @@ orderCommand.shippedOn.invalid.required=Propriété [{0}] de la classe [{1}] ave
 # Order Item messages
 orderItem.list.label=Liste d'articles en cours
 orderItem.missingGlAccount.label=Impossible d'ajouter un article de commande sans code de grand livre valide
-orderItem.warningSupplier.label=Cette source a le type de préférence "low quality" pour votre destination. Voulez-vous toujours l'ajouter au bon de commande?
+orderItem.warningSupplier.label=Cette source a le type de préférence \\"low quality\\" pour votre destination. Voulez-vous toujours l\\'ajouter au bon de commande?
```

Comment:
Look at the line above this. It seems like we're going to run into this a lot. So is it possible to do the escaping on the javascript side?

Reply:
@awalkowiak Thoughts?

Reply:
@awalkowiak If you guys figure this out don't worry about me reviewing, just merge it if it works.

Reply:
@awalkowiak @jmiranda I've figured it out - I replace all single quotes with \' to make sure it's displayed properly.